docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,166 @@
+
+After saving the entities, you will see a list of them in the detail view, where you can edit, add, or remove them.
+
+![](images/ts-12.png)
+
+## Publishing a Tracking Scenario
+
+Once you have finalized your definition of the tracking scenario and completed all necessary approval stages, you can publish it by clicking the **'Publish scenario'** button located at the right side of the scenario name in the detailed view.
+
+![](images/ts-13.png)
+
+Publishing a new scenario indicates an intention to implement the tracking defined in it. For data consumers, a published scenario signals that the tracking associated with the event complies with the rules contained in it.
+
+Once published, the scenario's status will appear as "Published" in the list view.
+
+If you need to edit a published scenario, you can do so by selecting the "Edit" button. This action will reset the scenario's status to "Draft."
+
+![](images/ts-14.png)
+
+## Sharing a Tracking Scenario
+
+If you want to share a tracking scenario, whether it is published or still a draft, follow these steps. First, go to the Tracking scenario tab. Then, click on the name of the tracking scenario you want to share. From the detailed view, click the **'Share scenario'** button, which is located near the publish button.
+
+![](images/ts-15.png)
+
+A dialog box will appear with a link and a **'Copy'** button.
+
+![](images/ts-16.png)
+
+## Deleting Draft Tracking Scenarios
+
+Whilst a tracking scenario is in *draft* mode, it can be deleted. To delete a draft tracking scenario, navigate to the **Tracking scenarios** tab and click the **'Delete'** button.
+
+A confirmation dialog box will appear.
+
+![](images/ts-17.png)
+
+:::warning
+This operation cannot be undone.
+:::
+
+## Deprecating a published Tracking Scenarios
+
+If you believe a published scenario no longer reflects your tracking implementation, or you wish to remove this tracking from your product, you can choose to deprecate it.
+
+Deprecating a tracking scenario signals the intention to remove this tracking from your product. It's important to note that deprecating a tracking scenario does not automatically prevent new events from being tracked in line with the rules of this scenario. Developers will need to manually stop tracking events for the deprecated scenario.
+
+Deprecating a scenario can also notify data consumers that the information contained in the scenario is not reflective of the tracking implementation and should not be used to understand the data.
+
+We strongly advise keeping your scenario in sync with your tracking, so that the information contained in it can be used by data consumers for data discovery and understanding purposes.
+
+To deprecate a published tracking scenario, go to the **Tracking scenarios** tab and click the **'Deprecate'** button. A confirmation dialog box will appear where you can add some comments.
+
+![](images/ts-18.png)
```

Comment:
We need to change the sub header to something like please inform the appropriate people of this deprecation, and update the screenshot

Reply:
https://snplow.atlassian.net/browse/XP-1751

Reply:
Updated